workflow/rocoto/workflow_tasks.py

```diff
@@ -1075,7 +1075,7 @@ def atmensanalpost(self):
 
     def ecen(self):
 
-        self._is_this_a_gdas_task(self.cdump, 'ecen')
+        #self._is_this_a_gdas_task(self.cdump, 'ecen')
```

Comment:
Just delete these instead of commenting them out.

Reply:
Thanks. Yes, I will delete them later.